src/python/pants/engine/legacy/graph.py

```diff
@@ -396,19 +396,16 @@ class TransitiveHydratedTargets:
 
 @dataclass(frozen=True)
 class SourcesSnapshot:
-  """A light wrapper around source files for Pants to operate on.
-
-  This corresponds 1-to-1 with the `sources` field of a target. When given filesystem specs, ...
-  TODO: what is the mapping for filesystem specs? Should each individual spec -> SourcesSnapshot?
-  Decide this once implementing the hydration of FilesystemSpecs to SourcesSnapshots."""
+  """A light wrapper around source files for Pants to operate on."""
```

Comment:
This description could be more useful... `SourcesSnapshot` should unambiguously mean something: maybe "sources matched by command line specs, either directly via FilesystemSpecs, or indirectly via AddressSpecs"?

Reply:
Sounds good. I'll go back to explaining how it maps to the two concepts. Thanks!